lib/mpl_toolkits/axes_grid1/axes_size.py

```diff
@@ -290,37 +290,21 @@ def get_size(self, renderer):
 
 
 class GetExtentHelper(object):
-    def _get_left(self, axes_bbox):
-        return axes_bbox.xmin - self.xmin
-
-    def _get_right(self, axes_bbox):
-        return self.xmax - axes_bbox.xmax
-
-    def _get_bottom(self, axes_bbox):
-        return axes_bbox.ymin - self.ymin
-
-    def _get_top(self, axes_bbox):
-        return self.ymax - axes_bbox.ymax
-
-    _get_func_map = dict(left=_get_left,
-                         right=_get_right,
-                         bottom=_get_bottom,
-                         top=_get_top)
-
-    del _get_left, _get_right, _get_bottom, _get_top
+    _get_func_map = {
+        "left":   lambda self, axes_bbox: axes_bbox.xmin - self.xmin,
```

Comment:
There was a problem with pickling of lambda functions recently. Is is ensured that the `GetExtentHelper` never needs to be pickled?

Reply:
The only place this is used is in axes_divider.Divider, which construcs a helper and immediately throws it away.  So it doesn't get store anywhere.

Reply:
Updated to not store the lambda on the instance.  Storing lambdas at the classlevel is fine re: pickling.